website/content/en/docs/deployment/rds-s3/guide.md

```diff
@@ -131,7 +131,7 @@ Follow this step if you prefer to manually set up each component.
       1. Rename the `parameters.objects.objectName` field in [the RDS Secret provider configuration](https://github.com/awslabs/kubeflow-manifests/blob/main/awsconfigs/common/aws-secrets-manager/rds/secret-provider.yaml) to the name of the Secret. 
          - Rename the field with the following command:
            ```bash
-           yq e -i '.spec.parameters.objects |= sub("rds-secret",env(RDS_SECRET))' awsconfigs/common/aws-secrets-manager/rds/secret-provider.yaml
+           yq e -i '.spec.parameters.objects |= sub("rds-secret",env($RDS_SECRET))' awsconfigs/common/aws-secrets-manager/rds/secret-provider.yaml
```

Comment:
This change is not valid. The current command is correct since it uses yq's `env` operator https://mikefarah.gitbook.io/yq/operators/env-variable-operators#read-string-environment-variable. This should work unless there is a bug in yq itself